stable/opsportal/values.yaml

```diff
@@ -114,3 +115,6 @@ kommander:
       traefik.ingress.kubernetes.io/auth-type: forward
       traefik.ingress.kubernetes.io/auth-url: http://traefik-forward-auth-kubeaddons.kubeaddons.svc.cluster.local:4181/
       traefik.ingress.kubernetes.io/auth-response-headers: X-Forwarded-User
+
+  kubefed:
```

Comment:
why does opsportal install kubefed ? shouldn't it be kommander chart who does it ?

Reply:
nvm! This is for the kommander chart.